cmd/enduro-a3m-worker/main.go

```diff
@@ -262,7 +246,7 @@ func main() {
 			temporalsdk_activity.RegisterOptions{Name: activities.ClassifySIPActivityName},
 		)
 		w.RegisterActivityWithOptions(
-			bagvalidate.New(validator).Execute,
+			bagvalidate.New(nil).Execute,
```

Comment:
@jraddaoui are you sure this is created a new validator on each workflow run? It looks to me like `bagvalidate.New()` will be run once, when the worker is started, and the same bagit_gython instance will be reused. :thinking: 

I would expect that we'd need to create and pass a new bagit_gython instance as an activity parameter in the workflow to get a new instance each run.

Reply:
It will reuse the same `gythonValidator`, but the bagit_gython instance is created when Validate is called.